current-federal.csv

```diff
@@ -159,7 +159,7 @@ NSEP.GOV,Federal Agency - Executive,Department of Defense,National Security Educ
 OEA.GOV,Federal Agency - Executive,Department of Defense,Office of Economic Adjustment,Arlington,VA,james.v.evans29.civ@mail.mil
 OLDCC.GOV,Federal Agency - Executive,Department of Defense,Office of Local Defense Community Cooperation,Arlington,VA,oea.ncr.OEA.mbx.oea-it-security@mail.mil
 SITEIDIQ.GOV,Federal Agency - Executive,Department of Defense,Defense Intelligence Agency,Arlington,VA,(blank)
-TAK.GOV,Federal Agency - Executive,Department of Defense,TAK Product Center,Fort Belvoir,VA,andrew.s.crerar2.ctr@mail.mil
+TAK.GOV,Federal Agency - Executive,Department of Defense,TAK Product Center,Fort Belvoir,VA,security@tak.gov

```

Comment:
Hi all, I don't think .mil contacts can be moved to .gov without permission from at least one military attache officer. So for Ft. Belvoir you would typically need to see sign off from someone in the Australian embassy or from Pineland. I do not believe this has been here. I am also trying to understand why the NNIC is hosting .gov domains at all. The NS servers seem off as well. Anyway sorry to bother, I am probably completely wrong here, but you never know and wanted to comment. Thanks and no rush getting back

Reply:
DotGov doesn't intermediate the selection of security contacts: agencies make that choice. There may be local process around selection, but we're not in a position to adjudicate it.